bazel/foreign_cc/BUILD

```diff
@@ -1,85 +1,101 @@
 licenses(["notice"])  # Apache 2
 
-load("//bazel:envoy_build_system.bzl", "envoy_package")
-load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
+load("//bazel:envoy_build_system.bzl", "envoy_cmake_external", "envoy_package")
 
 envoy_package()
 
-cmake_external(
+envoy_cmake_external(
     name = "ares",
     cache_entries = {
         "CARES_SHARED": "no",
         "CARES_STATIC": "on",
         "CMAKE_BUILD_TYPE": "RelWithDebInfo",
     },
-    cmake_options = ["-GNinja"],
+    copy_pdb = True,
     lib_source = "@com_github_c_ares_c_ares//:all",
-    make_commands = [
-        "ninja",
-        "ninja install",
-    ],
-    static_libraries = ["libcares.a"],
+    pdb_name = "c-ares",
+    static_libraries = select({
+        "//bazel:windows_x86_64": ["cares.lib"],
+        "//conditions:default": ["libcares.a"],
+    }),
 )
 
-cmake_external(
+envoy_cmake_external(
```

Comment:
Hmm, I missed this, why did you need to add this? repositories.bzl doesn't add the `:all` target to benchmark and it is built with upstream `BUILD.bazel` file. #5890

Reply:
Yeah, @sesmith177 could you followup on this one? Thanks.

Reply:
Yeah, that's definitely a mistake -- I hadn't noticed that when https://github.com/envoyproxy/envoy/commit/7fa55135828dc8d93c6eee6f363534afa5cb3484 removed the `benchmark.sh` script, it did not add a corresponding `cmake_external` rule. So when I merged our branch in to fix the Windows issues, this extra rule came with it.

Sorry for the confusion.